tests/sys/psa_crypto_mac/Makefile

```diff
@@ -1,13 +1,18 @@
 include ../Makefile.sys_common
 
+USEMODULE += embunit
+
 USEMODULE += ztimer
 USEMODULE += ztimer_usec
 
 USEMODULE += psa_crypto
-
 USEMODULE += psa_mac
+USEMODULE += psa_hash
```

Comment:
```suggestion
```

I guess this should be selected automatically if hmac is used?

Reply:
We need manual access to the hash functions for some tests, so i think explicitly selecting the psa_hash module is ok.

Reply:
In that case, you are right. :+1: